client/utils.go

```diff
@@ -17,3 +20,16 @@ const (
 func PipePath(apiGroupName string, apiVersion apiversion.Version) string {
 	return pipePrefix + csiProxyNamedPipePrefix + apiGroupName + "-" + apiVersion.String()
 }
+
+// FindFirstNamedPipe finds the first named pipe available at the Windows named pipe locations.
+// Only the first version from `apiVersions` that exists will be returned, this mechanism
+// allows picking the first version available in the case where there's a beta and v1 API
+func FindFirstNamedPipe(apiGroupName string, apiVersions []apiversion.Version) (string, error) {
```

Comment:
where this function is called?

Reply:
in the code that's using the client (in our case PD CSI), please check out the description of the PR

Reply:
I've removed this method and instead the scan is done in PD CSI